x/allocation/types/outgoing_tx.go

```diff
@@ -31,15 +32,14 @@ func (c Cellar) GetCheckpoint() []byte {
 		ticks = append(ticks, packTick{big.NewInt(0), big.NewInt(up), big.NewInt(lo), new(big.Int).SetUint64(we)})
 	}
 
-	// the methodName needs to be the same as the 'name' above in the checkpointAbiJson
-	// but other than that it's a constant that has no impact on the output. This is because
-	// it gets encoded as a function name which we must then discard.
 	abiEncodedCall, err := encodedCall.Pack("rebalance", ticks)
 	if err != nil {
 		panic(err)
 	}
 
-	return crypto.Keccak256Hash(abiEncodedCall).Bytes()
+	// TODO: why do I have to append the 0x in front here? Is this desirable? This is
+	// substantially different than the other calls
+	return crypto.Keccak256Hash([]byte(fmt.Sprintf("0x%x", abiEncodedCall))).Bytes()
```

Comment:
This is a bit unexpected, I feel like this is wrong, but fits with the hash we have been given. Investigate before merging